packages/openapi-typescript/src/lib/ts.ts

```diff
@@ -229,6 +229,8 @@ export function tsDedupe(types: ts.TypeNode[]): ts.TypeNode[] {
 export function tsEnum(
   name: string,
   members: (string | number)[],
+  membersNames?: string[],
```

Comment:
```suggestion
 metadata?: { name?: string, description?: string }[],
```

I think this is an easier-to-use signature; I find the former where multiple ordered params have the same type to be easy to make mistakes in rearranging them.

Reply:
Agree, done 👍